LayoutTests/imported/w3c/web-platform-tests/service-workers/service-worker/resources/testharness-helpers.js

```diff
@@ -96,28 +96,6 @@ function assert_array_objects_equals(actual, expected, description) {
     });
 }
 
-// Asserts that |object| that is an instance of some interface has the attribute
-// |attribute_name| following the conditions specified by WebIDL, but it's
-// acceptable that the attribute |attribute_name| is an own property of the
-// object because we're in the middle of moving the attribute to a prototype
-// chain.  Once we complete the transition to prototype chains,
-// assert_will_be_idl_attribute must be replaced with assert_idl_attribute
-// defined in testharness.js.
-//
-// FIXME: Remove assert_will_be_idl_attribute once we complete the transition
-// of moving the DOM attributes to prototype chains.  (http://crbug.com/43394)
-function assert_will_be_idl_attribute(object, attribute_name, description) {
```

Comment:
A PR needs to go upstream for this. Modifying it downstream won't be useful otherwise since it will get overwritten during the next resin.